ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -39,6 +39,8 @@
 #endif
 
 #include <stdio.h>
```

Comment:
if you include cstdio you shouldn't need stdio.h anymore. the "<cheader>" versions are just namespaced I think

Reply:
nice catch! I didn't actually notice the duplicate header at first.